source/includes/quick-start/create-project.rst

```diff
@@ -0,0 +1,11 @@
+This guide shows you how add the MongoDB Java driver dependencies using
+Maven or Gradle. We recommend that you use an integrated development
+environment (IDE) such as Intellij IDEA, Eclipse IDE, or NetBeans Java IDE to
```

Comment:
According to https://www.jetbrains.com/lp/devecosystem-2019/java/, NetBeans has lost.  Let's not even mention it.